solutions/swb-reference/end-to-end-tests/cypress/integration/common/environments.cy.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  CLEANUP_WAIT_ENVIRONMENTS_GRID_IN_MILISECONDS,
+  CLEAN_UP_ENVIRONMENTS,
+  ENVIRONMENT_PROJECT_PROPERTY,
+  ENVIRONMENT_START_MAX_WAITING_MILISECONDS,
+  ENVIRONMENT_STOP_MAX_WAITING_MILISECONDS,
+  ENVIRONMENT_STUDIES_PROPERTY,
+  ENVIRONMENT_TYPES_PROPERTY
+} from '../../support/constants';
+import { getFakeText, verifyDataGtid } from '../../support/common-utils';
+import { CreateEnvironmentForm, EnvTypeConfig } from '../../support/models';
+import {
+  cleanupEnvironments,
+  createEnvironment,
+  excecuteEnvironmentAction
+} from '../../support/environments-utils';
+
+describe('IT Admin Login', () => {
```

Comment:
`IT Admin Login` => `IT Admin`

These are test cases for `IT Admin`, and not specifically about IT Admin logging in.

Reply:
This is a copy paste mistake, I will rename it to Environment Operations

Reply:
Description changed